pkg/webhook/github.go

```diff
@@ -23,21 +23,23 @@ const (
 )
 
 type githubHook struct {
-	store        storage.Store
-	getFile      fileGetter
-	createStatus statusCreator
+	store                   storage.Store
+	getFile                 fileGetter
+	createStatus            statusCreator
+	buildForkedPullRequests bool
 }
 
 type fileGetter func(commit, path string, proj *brigade.Project) ([]byte, error)
 
 type statusCreator func(commit string, proj *brigade.Project, status *github.RepoStatus) error
 
 // NewGithubHook creates a GitHub webhook handler.
-func NewGithubHook(s storage.Store) *githubHook {
+func NewGithubHook(s storage.Store, buildForkedPullRequests bool) *githubHook {
 	return &githubHook{
-		store:        s,
-		getFile:      getFileFromGithub,
-		createStatus: setRepoStatus,
+		store: s,
+		buildForkedPullRequests: buildForkedPullRequests,
```

Comment:
is not the second argument, is the fourth right?

Reply:
This is for initializing a struct, not a function. Fields are not positional.